### PR TITLE
cellVdec: fix cellVdecClose wait loop

### PR DESCRIPTION
**Pull request by Megamouse**
No idea if this fixes anything or breaks everything xD

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellVdec.cpp [hunk 1]
```diff
@@ -924,7 +924,7 @@ error_code cellVdecClose(ppu_thread& ppu, u32 handle)
 	vdec->abort_decode = true;
 	vdec->in_cmd.push(vdec_cmd(vdec_cmd_type::close, seq_id, cmd_id));
 
-	while (!vdec->ppu_tid)
+	while (!vdec->ppu_tid || vdec->seq_state != sequence_state::closed)
```
Comment (elad335) [inline rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: Not a bug, this code waits to obtain ppu_tid which is zero initialized and set by VDEC thread.

**Review by elad335 — CHANGES_REQUESTED** [review]

File: rpcs3/Emu/Cell/Modules/cellVdec.cpp (hunk 1 above)
Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: How does it relate to what I added?

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: ppu_tid is always written eventually, no need in another condition.

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: ppu_tid is written at thread start.
I want to wait for thread end

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: That's what the syscall is for.

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: I assume the syscall is setting thread abort.
Which is different than abort_decode.

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: No it joins the thread.

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: ah ok. thx

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellVdec.cpp:927]: but line 940 is redumdant indeed.

**Pull request closed without merging**